### DISCUSSION TITLE
New boards are unusable due not being "repo first"

### DISCUSSION THREAD

**Discussion by crutkas** [community/community#10318] [Projects and Issues]
If i can't quickly add / move / remove issues in both the board and issues themselves, the new system is unusable.  Having to hunt for a project is just too much in a project.  I lost too much speed trying to setup a new project and just put stuff into it.  I can't just do #ISSUE_NUMBER

For Microsoft/PowerToys, we have milestones as things like backlog or "priority work" buckets and projects are the current sprints.

In current state, i have to grant access then to the board versus inheriting everything from the repo.

The TLDR IMO should be the board MUST act like the repo.  If i can't do the same actions, it is introducing concepts force context shifts.
